doc/sphinx/fakedbusdoc.py

```diff
@@ -0,0 +1,25 @@
+# D-Bus XML documentation extension, compatibility gunk for <sphinx4
```

Comment:
Do we need this fake file? If so maybe a a bit more reasoning why would be good to have in the comment.

Reply:
If we use sphinx > 4 we can drop it. we don't specify any minimum version of sphinx. should we?

Reply:
Sphinx 4 is quite new thing. It was released in F35. But on the other hand we are not able to build dnf5 on earlier Fedoras because of rpm 4.17 requirement (and maybe also c++ 20?). So I wouldn't be that afraid of bumping sphinx requirement to 4.